src/proof/lfsc_proof_printer.h

```diff
@@ -74,7 +74,35 @@ class LFSCProofPrinter
                                          std::ostream& out,
                                          std::ostream& paren);
 
+  /**
+   * The SAT solver is given a list of clauses.
+   * Assuming that each clause has alreay been individually proven,
+   * defines a proof of the input to the SAT solver.
+   *
+   * Returns the name of the LFSC value holding the proof, i.e. a value of type
+   * (clauses_hold ...)
```

Comment:
where is this type defined?

Reply:
This is a _very_ good catch! You're right that no such type is defined.

I don't actually want to be printing a proof that some `cnf` holds, I want to be printing a proof that some `CMap` holds.

I think that at some point I was using the `clauses_hold` type, but I've since removed it.

I would have discovered this once I actually tried to put the pieces together, but all the better to discover it now!

Thanks!